.github/workflows/auto-assign.yml

```diff
@@ -2,6 +2,9 @@ name: Auto Assign
 on:
   pull_request:
     types: [opened, reopened]
+permissions:
```

Comment:
I think we need `pull-request: write` here because `gh-auto-assign` assigns PRs to users